src/app/app.component.ts

```diff
@@ -1,9 +1,10 @@
 import { Component } from '@angular/core';
 
 @Component({
-	selector: 'app-root',
-	templateUrl: './app.component.html',
-	styleUrls: ['./app.component.scss']
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss'],
```

Comment:
Let's loose the trailing commas where not necessary. There used to be a lint rule for that. Not sure what happened to it.